doc/stdlib/parallel.rst

```diff
@@ -234,6 +234,18 @@ General Parallel Computing Support
 
    Fetch the value of a remote reference, removing it so that the reference is empty again.
 
+       procList = addprocs(1)              #  add another process
+       rr  =  RemoteRef(procList[1])       #  create a remote reference on the next process
+       @everywhere function f(inRR::RemoteRef)
+         x  =  take!(inRR)
+         put!(inRR, x ^ 2)
+       end
+       @spawnat procList[1] f(rr)
+       put!(rr, 8)
+       @show( take!(rr) )  #   =>  64
```

Comment:
Though the chances of seeing a return value of 8 here is almost nil, julia does not guarantee that `f` will execute on the remote worker before the `take!` here.

Hence, a better example for message passing would be to have two remote refs, say r1 and r2. r1 is always written to from the local process and read from the remote process, while r2 is always written to at the remote process and read from the local.